### DISCUSSION TITLE
Calling All Potential Collaborators!

### DISCUSSION THREAD

**Discussion by hollasch** [RayTracing/raytracing.github.io#1218] [Meta]
This is just a general discussion for people who are interested in helping out with the project in some way. I've just tagged a work issue with our "Help Wanted" tag, and will see if there are other issues that we can easily hand off to new contributors. Here's the query:

https://github.com/RayTracing/raytracing.github.io/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22

We'd love to have additional contributors, though the particular tasks are (and likely always will be) a bit fuzzy. Don't be afraid to reach out, either in this thread, or to me directly (steve@hollasch.net).

**Comment by dimitry-ishenko** [comment]
I only see one issue tagged as "Help Wanted". Will there be others?

**Reply by hollasch** [reply]
There should be. I'll make a quick pass to see what makes sense.

Is there something you'd be interested in helping with?

**Reply by dimitry-ishenko** [reply]
I'd be up for some code review and clean-up.
